ui/component/channelCreate/view.jsx

```diff
@@ -0,0 +1,192 @@
+// @flow
```

Comment:
We already have the `SelectChannel` component. We should be able to re-use most of that code here.